src/common/event-emitter/contextual-ee.js

```diff
@@ -34,7 +33,7 @@ function ee (old, debugId) {
     // We only want to check the runtime configuration for `isolatedBacklog` if the event emitter belongs to a feature.
     // For feature event emitters, the debugId will be an agentIdentifier with a length of 16. In contrast, some of our
     // tests do not namespace the event emitter with an agentID and just use the parent (which has no ID).
-    isolatedBacklog = debugId.length !== 16 ? false : getRuntime(debugId).isolatedBacklog
+    isolatedBacklog = debugId.length !== 16 ? false : nr.initializedAgents?.[debugId]?.isolatedBacklog
```

Comment:
this should probably have a fallback of `false` if it fails to find `initializedAgents` -- though it likely doesnt matter much if its just used as a truthy check and not explicitly checking for `false` so that just needs to be checked

Reply:
Right, no usage of isolatedBacklog does an explicit check for `false`, so the fallthrough to undefined in that case works fine.

Reply:
I did figure out this is causing the test failure because it's missing the `.runtime` but that's another issue... heh